src/PropertyDocumentMixin.js

```diff
@@ -280,7 +282,20 @@ const mxFunction = (base) => {
           return [item];
         default: {
           const pkey = this._getAmfKey(this.ns.w3.shacl.property);
-          const items = this._ensureArray(item[pkey]);
+          let items = this._ensureArray(item[pkey]);
+
+          if (!items) {
+            const skey = this._getAmfKey(this.ns.w3.rdfSchema.Seq);
+            if (this._hasType(item, skey)) {
+              const rkey = `${this._getAmfKey(this.ns.w3.rdfSchema.key)}_1`;
```

Comment:
I'm a little afraid of the `_1` in this string. Won't it be a problem? Is it possible there will be another number here?

Reply:
You're right. Changed it so that it looks for properties in schemas starting with _http://www.w3.org/2000/01/rdf-schema#__